spec/fixtures/sample_number_as_name.yml

```diff
@@ -0,0 +1,41 @@
+# ~/.tmuxinator/sample_number_as_name.yml
+# you can make as many tabs as you wish...
+
+name: 222
```

Comment:
Would you be willing to pair this fixture down to a more bare bones tmux project file?  I think it will better isolate what your test is operating on as well as make it less likely for future changes to the parsing of the other yaml options break your test.